spec/index.bs

```diff
@@ -2306,6 +2324,71 @@ namespace GPUBufferUsage {
 };
 </script>
 
+The {{GPUBufferUsage}} flags determine how a {{GPUBuffer}} may be used after it's creation:
+
+<dl dfn-type="const" dfn-for=GPUBufferUsage>
+    : <dfn>MAP_READ</dfn>
+    ::
+        The buffer can be mapped for reading. (Example: calling {{GPUBuffer/mapAsync()}} with
+        {{GPUMapMode/READ|GPUMapMode.READ}})
+
+        May only be combined with {{GPUBufferUsage/COPY_DST}}.
+
+    : <dfn>MAP_WRITE</dfn>
+    ::
+        The buffer can be mapped for writing. (Example: calling {{GPUBuffer/mapAsync()}} with
+        {{GPUMapMode/WRITE|GPUMapMode.WRITE}})
+
+        May only be combined with {{GPUBufferUsage/COPY_SRC}}.
+
+    : <dfn>COPY_SRC</dfn>
+    ::
+        The buffer can be used as the source of a copy operation. (Examples: as the `source`
+        argument of a {{GPUCommandEncoder/copyBufferToBuffer()}} or
+        {{GPUCommandEncoder/copyBufferToTexture()}} call.)
+
+    : <dfn>COPY_DST</dfn>
+    ::
+        The buffer can be used as the destination of a copy or write operation. (Examples: as the
+        `destination` argument of a {{GPUCommandEncoder/copyBufferToBuffer()}} or
+        {{GPUCommandEncoder/copyTextureToBuffer()}} call, or as the target of a
+        {{GPUQueue/writeBuffer()}} call.)
+
+    : <dfn>INDEX</dfn>
+    ::
+        The buffer can be used as an index buffer. (Example: passed to
+        {{GPURenderCommandsMixin/setIndexBuffer()}}.)
+
+    : <dfn>VERTEX</dfn>
+    ::
+        The buffer can be used as a vertex buffer. (Example: passed to
+        {{GPURenderCommandsMixin/setVertexBuffer()}}.)
+
+    : <dfn>UNIFORM</dfn>
+    ::
+        The buffer can be used as a uniform buffer. (Example: as a bind group entry for a
+        {{GPUBufferBindingLayout}} with a
+        {{GPUBindGroupLayoutEntry/buffer}}.{{GPUBufferBindingLayout/type}} of
+        {{GPUBufferBindingType/"uniform"}}.)
+
+    : <dfn>STORAGE</dfn>
+    ::
+        The buffer can be used as a storage buffer. (Example: as a bind group entry for a
+        {{GPUBufferBindingLayout}} with a
+        {{GPUBindGroupLayoutEntry/buffer}}.{{GPUBufferBindingLayout/type}} of
+        {{GPUBufferBindingType/"storage"}} or {{GPUBufferBindingType/"read-only-storage"}}.)
+
+    : <dfn>INDIRECT</dfn>
+    ::
+        The buffer can be used as to store indirect command arguments. (Examples: as the
+        `indirectBuffer` argument of a {{GPURenderCommandsMixin/drawIndirect()}} or
+        {{GPUComputePassEncoder/dispatchWorkgroupsIndirect()}} call.)
+
+    : <dfn>QUERY_RESOLVE</dfn>
+    ::
+        The buffer can be used to capture query results.
```

Comment:
nit: Example should link to resolveQuerySet

Reply:
Ah, sorry I missed that one somehow! Fixed.